include/tesla.hpp

```diff
@@ -3498,6 +3668,8 @@ extern "C" {
      */
     void __appInit(void) {
         tsl::hlp::doWithSmSession([]{
+            ASSERT_FATAL(capsscInitialize());
```

Comment:
I don't know if I like using up even more sessions. How tight is the limit on them? Maybe rather move these two into the screenshot function

Reply:
There are 4 sessions.
Currently am and grc occupy one each.
Creport stock uses one on debug but since we aren't on debug and AMS' creport doesn't use it it's free.

Reply:
*Creport also only occupies it when a crash happens.

Reply:
*On debug so it doesn't